controllers/storagecluster/cephfilesystem.go

```diff
@@ -121,11 +122,33 @@ func (obj *ocsCephFilesystems) ensureCreated(r *StorageClusterReconciler, instan
 				return reconcile.Result{}, err
 			}
 		}
+		// create default csi subvolumegroup for the filesystem
+		err = r.createDefaultCsiSubvolumeGroup(cephFilesystem.Name, cephFilesystem.Namespace)
+		if err != nil {
+			return reconcile.Result{}, err
+		}
 	}
 
 	return reconcile.Result{}, nil
 }
 
+func (r *StorageClusterReconciler) createDefaultCsiSubvolumeGroup(filesystemName, filesystemNamespace string) error {
+	objectMeta := metav1.ObjectMeta{Name: "csi", Namespace: filesystemNamespace}
+	cephFilesystemSubVolumeGroup := &cephv1.CephFilesystemSubVolumeGroup{ObjectMeta: objectMeta}
+	mutateFn := func() error {
+		cephFilesystemSubVolumeGroup.Spec = cephv1.CephFilesystemSubVolumeGroupSpec{
+			FilesystemName: filesystemName,
+		}
+		return nil
+	}
+	_, err := ctrl.CreateOrUpdate(r.ctx, r.Client, cephFilesystemSubVolumeGroup, mutateFn)
+	if err != nil {
+		r.Log.Error(err, "Could not create default csi cephFilesystemSubVolumeGroup.", "cephFilesystemSubVolumeGroup", klog.KRef(cephFilesystemSubVolumeGroup.Namespace, cephFilesystemSubVolumeGroup.Name))
+		return err
+	}
+	return nil
+}
```

Comment:
SVG also need to be deleted before filesystem in `ensureDeleted` function?

Reply:
Added